Misc/NEWS.d/next/Library/2019-03-14-09-08-25.bpo-35859.8lFdLe.rst

```diff
@@ -0,0 +1,2 @@
+re module, fix a few bugs about capturing group. In rare cases, capturing
+group gets an incorrect string. Patch by Ma Lin.
```

Comment:
Docs tests are failing due to a lack of newline at the end of the NEWS entry

```suggestion
group gets an incorrect string. Patch by Ma Lin.

```

Reply:
Fixed.